cmd/user-agent/src/pages/DIDManagement.vue

```diff
@@ -233,6 +235,16 @@ SPDX-License-Identifier: Apache-2.0
                     friendlyName: this.friendlyName
                 })
 
+                // saving did to SDS
```

Comment:
should wallet DID needs to be stored in SDS?

Reply:
@sudeshrshetty yes

Reply:
then, that code is missing in your PR @DRK3

Reply:
@sudeshrshetty Do you mean the "save any DID" feature? If so... yes indeed you are correct! I will add that now.

(Let me know if you meant something else)

Reply:
@sudeshrshetty, just to clarify, I do save the DID to EDV, but only the ones created in the DID dashboard. Not sure if you thought that there wasn't any DID -> EDV saving going on.

Reply:
(done - save any did now saves to EDV as well)

Reply:
@DRK3 I was talking about wallet DID, the one you changed in this file is for DID management demo page.
Wallet registration creates DID which gets stored in aries.vdri (local store). I think you should change that code too..

Ref: https://github.com/trustbloc/edge-agent/blob/master/cmd/user-agent/src/pages/chapi/wallet/register/register.js#L39

Reply:
@sudeshrshetty I've added it now - now the save DID to EDV logic is inside that SaveDID function, so whoever calls it will get the DID saved in the local storage and SDS - if you refresh the file changes you should see it there. Let me know if it still looks like it's missing